test/helper.lua

```diff
@@ -118,4 +237,28 @@ function helpers.is_expired_debug(_, tuple)
     return true
 end
 
+function helpers.tarantool_version()
+    local major_minor_patch = _G._TARANTOOL:split('-', 1)[1]
+    local major_minor_patch_parts = major_minor_patch:split('.', 2)
+
+    local major = tonumber(major_minor_patch_parts[1])
+    local minor = tonumber(major_minor_patch_parts[2])
+    local patch = tonumber(major_minor_patch_parts[3])
+
+    return major, minor, patch
+end
+
+function helpers.vinyl_is_broken()
+    -- Blocked by https://github.com/tarantool/tarantool/issues/6448
+    local major, minor, patch = helpers.tarantool_version()
+
+    -- Since Tarantool 1.10.11.
+    local broken_v1_10 = major >= 1 and (minor > 10 or minor == 10 and patch >= 11)
+
+    -- Tarantool >= 2.1.7 and < 2.8.2.
```

Comment:
There was no 2.1.7 version

Reply:
The latest version is [2.1.3](https://github.com/tarantool/tarantool/releases/tag/2.1.3) is and first one is [2.1.0](https://github.com/tarantool/tarantool/releases/tag/2.1.0).
Updated a comment.